### PR TITLE
[UPDATE] Readme - Adding table of contents on readme

### PR DESCRIPTION
**Pull request by Zakichanu**
### Problem

There wasn't a table of contents to jump right into chapters
### Solution

Add table of contents chapter

### Result

We will have the possibility to get into a chapter quickly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,26 @@ Twitter API v2 available endpoints
...
+## Table of contents
+
+- [Requirements](#requirements)
+- [Installation](#installation)
+  - [Maven users](#maven-users)
+  - [Gradle users](#gradle-users)
+  - [Others](#others)
+- [Twitter Credentials](#twitter-credentials)
+  - [OAuth 2.0 Authorization with PKCE](#oauth-20-authorization-with-pkce)
+  - [OAuth 2.0 Bearer Token](#oauth-20-bearer-token-app-only)
```
Comment (zacmos) [inline README.md:25]: It looks that the "OAuth 2.0" links don't work.
I think that it should be: 
```
(#oauth-20-authorization-with-pkce-)
(#oauth-20-bearer-token-app-only-)
```

File: README.md [hunk 2]
```diff
@@ -79,7 +99,7 @@ Then manually install the following JARs:
 
 Twitter APIs support three types of authentications:
 
-* OAuth 2.0 Authorization with PKCE
+* #### OAuth 2.0 Authorization with PKCE
```
Comment (Zakichanu) [inline README.md:102]: In the first commit, I put a colon after the little title, I removed and it might work now

File: README.md [hunk 3]
```diff
@@ -91,7 +111,7 @@ TWITTER_OAUTH2_REFRESH_TOKEN
 
 TWITTER_OAUTH2_IS_AUTO_REFRESH_TOKEN - default value is `false`
 
-* OAuth 2.0 Bearer Token (app-only)
+* #### OAuth 2.0 Bearer Token (app-only)
```
Comment (Zakichanu) [inline README.md:114]: In the first commit, I put a colon after the little title, I removed and it might work now

**Review by zacmos — APPROVED** [review]

**Comment by zacmos** [comment]
Thank you @Zakichanu

**Pull request merged**